radicle-cli/src/commands/issue.rs

```diff
@@ -399,16 +401,96 @@ pub fn run(options: Options, ctx: impl term::Context) -> anyhow::Result<()> {
     Ok(())
 }
 
-fn show_issue(issue: &issue::Issue) -> anyhow::Result<()> {
+// FIXME: this is splitting a long string into a small chunks,
+// only because our UI component is not take into count this case,
+// and we are splitting the line by `\n` and not by string length!
+//
+// We should fix the UI component, and remove the function here
+fn sub_strings(string: &str, sub_len: usize) -> Vec<&str> {
+    let mut subs = Vec::with_capacity(string.len() / sub_len);
+    let mut iter = string.chars();
+    let mut pos = 0;
+
+    while pos < string.len() {
+        let mut len = 0;
+        for ch in iter.by_ref().take(sub_len) {
+            len += ch.len_utf8();
+        }
+        subs.push(&string[pos..pos + len]);
+        pos += len;
+    }
+    subs
+}
+
+fn show_issue(whoami: &NodeId, issue: &issue::Issue) -> anyhow::Result<()> {
     let tags: Vec<String> = issue.tags().cloned().map(|t| t.into()).collect();
-    let assignees: Vec<String> = issue.assigned().map(|a| a.to_string()).collect();
+    let assignees: Vec<String> = issue
+        .assigned()
+        .map(|a| {
+            if a == whoami.into() {
+                "You".to_owned()
+            } else {
+                a.to_string()
+            }
+        })
+        .collect();
+
+    let mut attrs = Table::<2, Paint<String>>::new(TableOptions {
+        spacing: 2,
+        ..TableOptions::default()
+    });
+
+    attrs.push([
+        term::format::tertiary("Title".to_owned()),
+        term::format::bold(issue.title().to_owned()),
+    ]);
+
+    if !tags.is_empty() {
+        attrs.push([
+            term::format::tertiary("Tags".to_owned()),
+            term::format::secondary(tags.join(", ")),
+        ]);
+    }
+
+    if !assignees.is_empty() {
+        attrs.push([
+            term::format::tertiary("Assignees".to_owned()),
+            term::format::dim(assignees.join(", ")),
+        ]);
+    }
 
-    term::info!("title: {}", issue.title());
-    term::info!("state: {}", issue.state());
-    term::info!("tags: [{}]", tags.join(", "));
-    term::info!("assignees: [{}]", assignees.join(", "));
+    attrs.push([
+        term::format::tertiary("Status".to_owned()),
+        if issue.is_open() {
+            term::format::positive("Open".to_owned())
+        } else {
+            match issue.state() {
+                issue::State::Closed { reason } => term::format::default(format!(
+                    "{} {}",
+                    term::format::negative("Closed"),
+                    term::format::default(format!("as {reason}"))
+                )),
+                _ => anyhow::bail!("state closed expected"),
```

Comment:
I meant `match` like this:
```suggestion
        match issue.state() {
            issue::State::Open => term::format::positive("Open".to_owned())
            issue::State::Closed { reason } => term::format::default(format!(
                    "{} {}",
                    term::format::negative("Closed"),
                    term::format::default(format!("as {reason}"))
                )),
        }
```